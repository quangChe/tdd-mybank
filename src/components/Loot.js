import React from 'react';

export class Loot extends React.Component {
  render() {
    return (
      <h3>Bitcoin balance: </h3>
    )
  }
}

export default Loot;