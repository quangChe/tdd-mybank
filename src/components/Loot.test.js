import React from 'react';
import { shallow } from 'enzyme';
import { Loot } from './Loot';

describe('Loot', () => {
  const loot = shallow(<Loot />);

  it('renders correctly', () => {
    expect(loot).toMatchSnapshot();
  })
})
