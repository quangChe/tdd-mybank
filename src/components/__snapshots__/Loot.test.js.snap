// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loot renders correctly 1`] = `
<h3>
  Loot Component
</h3>
`;
